v2/cli/term/event_page.go

```diff
@@ -35,14 +37,28 @@ func newEventPage(
 		eventInfo:  tview.NewTextView().SetDynamicColors(true),
 		workerInfo: tview.NewTextView().SetDynamicColors(true),
 		jobsTable:  tview.NewTable().SetSelectable(true, false),
+		eventLogs:  tview.NewTextView().SetDynamicColors(true),
 		usage: tview.NewTextView().SetDynamicColors(true).SetText(
-			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
+			"[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](L) [white]Logs    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit", // nolint: lll
 		),
 	}
 	e.eventInfo.SetBorder(true).SetBorderColor(tcell.ColorYellow)
 	e.workerInfo.SetBorder(true).SetBorderColor(tcell.ColorYellow).
 		SetTitle("Worker")
 	e.jobsTable.SetBorder(true).SetTitle("Jobs")
+	e.eventLogs.SetBorder(true).SetTitle("Logs (<-/Del) Quit")
+
+	// Returns a new primitive which puts the provided primitive in the center and
+	// sets its size to the given width and height.
+	e.logModal = tview.NewFlex().
+		AddItem(nil, 0, 1, false).
```

Comment:
For my own edification-- are the several `AddItem(nil, 0, 1, false)` all about putting padding the layout with dead space so the page appears as a floating window? This is fine. I just want to make sure I understand correctly. I am no expert with this UI toolkit.

Reply:
Yes, this is to make the textfield appear to be a floating window